### ISSUE TITLE
Project alive?

### ISSUE THREAD

**Issue by dnorthup-ums** [janpascal/denyhosts_sync#36]
@janpascal and others,
Is this project still considered "alive"? Is it Python 3 ready?

**Comment by janpascal** [comment]
Yes, it is certainly alive and used in production, although there has been no recent development. I'm not sure about Python 3 compatibility. I remember taking it into account, but it's been a while.

dnorthup-ums <notifications@github.com> schreef op 5 maart 2020 15:43:03 CET:
>@janpascal and others,
>Is this project still considered "alive"? Is it Python 3 ready?
>
>-- 
>You are receiving this because you were mentioned.
>Reply to this email directly or view it on GitHub:
>https://github.com/janpascal/denyhosts_sync/issues/36

-- 
Verstuurd vanaf mijn Android apparaat met K-9 Mail. Excuseer mijn beknoptheid.
